base/deprecated.jl

```diff
@@ -1086,4 +1087,6 @@ eval(Base.LinAlg, quote
     end
 end)
 
+Filesystem.stop_watching(stream::Filesystem._FDWatcher) = depwarn("stop_watching(::_FDWatcher) is removed", :stop_watching)
```

Comment:
it's not just deprecated, it's removed (replaced by a warning that hopes the theoretical, non-existant caller wasn't expecting a side effect)